sdk/java-sdk/src/main/scala/kalix/javasdk/impl/action/ActionEffectImpl.scala

```diff
@@ -81,6 +85,10 @@ object ActionEffectImpl {
       AsyncEffect(futureMessage.asScala.map(s => Builder.reply[S](s))(ExecutionContext.parasitic), Nil)
     def asyncEffect[S](futureEffect: CompletionStage[Action.Effect[S]]): Action.Effect[S] =
       AsyncEffect(futureEffect.asScala, Nil)
+    def empty[S](): Action.Effect[S] =
```

Comment:
I think we should skip the `empty` and only have `ignore`

Reply:
I thought `empty` for an endpoint usage. Mainly because `ignore` doesn't convey meaning there. Ignore makes sense on a stream but I can't quite make sense out of it in a `resquest/response` scenario. As a user, if a want to say this action gives no effect back. Don't you think that using ignore implies a knowledge of how things work underneath? Maybe we can use `noReply` instead?

Reply:
ok then. I'll leave it to the day we have a pure Spring SDK 👍